src/Sarif.UnitTests/Core/FileDataTests.cs

```diff
@@ -34,7 +33,7 @@ public void FileData_ComputeHashes()
             {
                 File.WriteAllText(filePath, fileContents);
                 FileData fileData = FileData.Create(uri, LoggingOptions.ComputeFileHashes);
-                fileData.Uri.Should().Be(uri.ToString());
+                fileData.Uri.Should().BeNull();
```

Comment:
>fileData.Uri [](start = 16, length = 12)

file.Uri is actually something like file.NestedUri or file.RelativeUri (for cases when an item is relative to some container file).

Reply:
Not necessarily. See my other comment.

---
In reply to: [136618490](https://github.com/Microsoft/sarif-sdk/pull/614#discussion_r136618490) [](ancestors = 136618490)